app/containers/App/selectors.js

```diff
@@ -111,7 +116,20 @@ const makeEntitySelector = () => createSelector(
 
 const makeEntityMapSelector = () => createSelector(
```

Comment:
I think this makes sense to have in App/selectors

I'm thinking that what you are doing here should be the default case when we get entities back from the selector, as this is really what we are going to be wanting to use most of the time, we shouldn't have to think about joining the entities up, that should be the default case ready to use.

naming wise `completeEntitiesSelector`  or `joinedEntitiesSelector` or simply `entitiesSelector` with the non joined entities not exposed at all via a selector... :thinking: